tests/macos/xcodeproj/Single-Application-Project-AllTargets.xcodeproj/bazelinstallers/indexstores.sh

```diff
@@ -4,11 +4,23 @@ set -euo pipefail
 
 # See `_indexstore.sh` for full details.
 
-# Make sure add these to bazel build target copts for objective-c
-find "$BAZEL_WORKSPACE_ROOT/bazel-out"/*/bin/ \
-     -type d \
-     -name "*.indexstore" \
-     -print0 \
-    | xargs -0 "$BAZEL_INSTALLERS_DIR/_indexstore.sh"
-
-echo "Finish remapping index files"
\ No newline at end of file
+echo "Start remapping index files at `date`"
```

Comment:
debugging?

Reply:
I put that in here intentionally, because it made it easy for me to look inside the bazel-xcode-diagnostics directory to see if my index remapping was in progress.